src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActions.tsx

```diff
@@ -77,6 +78,23 @@ export class ArtworkActions extends React.Component<
     }
   }
 
+  openViewInRoom(mediator) {
+    const {
+      artwork: { dimensions, image },
+    } = this.props
+    mediator &&
+      mediator.trigger &&
+      mediator.trigger("openViewInRoom", {
+        dimensions,
+        image,
+      })
+
+    // Still to do:
+    // - Use correct eye button (being updated in palette; we might not even have to change the code if we're lucky)
+    // - Define mediator behavior in Force
+    // - Fix unhappy onClick (line 121)
```

Comment:
What is 'unhappy onClick'?

Reply:
For the force side - I think we need to look what it actually does with the image. Maybe we can get away with modifying code a bit and based on ImageId - fetch the image and pass it to view in a room.

Reply:
Ah sorry for being unspecific about the onClick issue—I'm seeing this error from typescript: [screenshot](https://cl.ly/d6e32ab36ef8)

Reply:
As discussed in person - it expects a function itself where we are doing a direct function call which just returns void. I thing we got it corrected:) So I think we can merge that once it's all green.

Reply:
yep! just pushed a commit with that fix. tests and storybook look good (except for the old eye icon)—note that to see it in the storybook, you'll need to remove the is_hangable check

Reply:
@mdole - We actually need to add proper test case and update the fixture to show it in storybooks. I'm updating this file slightly so will deal with that as well.